song-server/src/main/java/bio/overture/song/server/service/FileModificationService.java

```diff
@@ -100,39 +123,29 @@ public FileUpdateResponse securedFileWithAnalysisUpdate(
             + "be updated since its analysisState is '%s'",
         objectId,
         analysisId,
-        SUPPRESSED.toString());
+        currentState.toString());
 
     // Update the target file record using the originalFile and the update request
     val fileUpdateType = updateWithRequest(originalFile, fileUpdateRequest);
-
-    // Build the response
     val response = FileUpdateResponse.builder().unpublishedAnalysis(false);
+    if (fileUpdateType == METADATA_UPDATE || fileUpdateType == CONTENT_UPDATE) {
+      response.message(
+          format("Updated file with objectId '%s' and analysisId '%s'", objectId, analysisId));
+    } else if (fileUpdateType == NO_UPDATE) {
+      response.message(
+          format(
+              "No update for file with objectId '%s' and analysisId '%s'", objectId, analysisId));
+    } else {
+      throw new IllegalStateException("Could not process fileUpdateType: " + fileUpdateType);
+    }
+    // Build the response
+
     response.originalFile(fileConverter.convertToFileDTO(originalFile));
     response.originalAnalysisState(currentState);
     response.fileUpdateType(fileUpdateType);
+    response.message(
+        format("Updated file with objectId '%s' and analysisId '%s'", objectId, analysisId));
```

Comment:
this should be deleted. Once this is deleted, it will affect the tests i commented on below